infra/build/functions/target_experiment.py

```diff
@@ -213,17 +244,25 @@ def main():
   parser.add_argument('--upload_corpus',
                       required=True,
                       help='GCS location to upload corpus.')
+  parser.add_argument('--upload_reproducer',
+                      required=True,
```

Comment:
Should this be optional?  This way, migrating oss-fuzz-gen will be seamless. Otherwise, as soon as this is submitted, oss-fuzz-gen will break in the interim before we add this flag there.

Reply:
Yes, good point.
I will make it optional.

Reply:
Fixed